<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeCategoryUI</class>
 <widget class="QDialog" name="ChangeCategoryUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Category:</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Current categories for </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="lstCategories">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add Category</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbExisting">
        <property name="text">
         <string>Choose existing</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="txtNewCategory"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbCategories"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbNew">
        <property name="text">
         <string>Create new</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>208</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="cmdAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>cbCategories</zorder>
     <zorder>txtNewCategory</zorder>
     <zorder>cmdAdd</zorder>
     <zorder>horizontalSpacer_2</zorder>
     <zorder>rbNew</zorder>
     <zorder>rbExisting</zorder>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
